src/apply.c

```diff
@@ -5,8 +5,6 @@
  * a Linking Exception. For full terms see the included COPYING file.
  */
 
-#include "apply.h"
-
```

Comment:
We have the rule that a given implementation should always include its own header first. 
This is mainly to make sure that we've got the include chain correct: apply.c -> apply.h -> common.h. With the changed include, we now have our public headers first and are thus missing "common.h" as first include.

This is another thing I should probably have in our coding style document.